### PR TITLE
Updated tutorial part 2 for clarity

### PR DESCRIPTION
**Pull request by dogeared**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

Issue Number: #19714

## What is the new behavior?
Adds a little descriptive text in the `Expose heroes` section of part 2 of the tutorial

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/tutorial/toh-pt2.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Eventually this app will fetch the list of heroes from a web service, but for no
 you can display mock heroes.
 
 ### Expose heroes
-Create a public property in `AppComponent` that exposes the heroes for binding.
+Create a public property in `AppComponent` (between the existing `title` and `hero` properties) that exposes the heroes for binding.
```
Comment (gkalpak) [inline aio/content/tutorial/toh-pt2.md]: As discussed in #19714, the order of properties doesn't make any difference. This change makes it sound as if there are more constraints than there actually are. IMO _"create a public property in `AppComponent`"_ is adequately describing what needs to be done.

Comment (jenniferfell) [reply aio/content/tutorial/toh-pt2.md]: @kapunahelewong and I were just discusssing. What about something like this: Create a public property in 'AppComponent' that exposes the heroes for binding. To make it easier to read your code, keep all public properties together. For this tutorial, define [create? put?] the 'heroes' property between the existing 'title' and 'hero' properties.

Comment (kapunahelewong) [reply aio/content/tutorial/toh-pt2.md]: @dogeared We could also expand the docregion for the example code to show it in context.

**Comment by dogeared** [comment]
No disagreement on order. In other places, the tutorial is explicit about
where to place things as it makes it more clear. For instance:
https://angular.io/tutorial/toh-pt1#hero-object

It's not technically critical that the new Hero class goes right below the
import statement. But, it does make it more clear for Angular newbies.

==============================================
Micah Silverman, CSM <http://bit.ly/WikiScrum>
Phone: 631.606.8928
Co-Author: Mastering Enterprise Javabeans 3.0 <http://bit.ly/MasteringEJB3>
==============================================
"Debugging is twice as hard as writing the code in the first
place. Therefore, if you write the code as cleverly as
possible, you are, by definition, not smart enough to debug
it." by Brian Kernighan
==============================================
[image: Namez]
<http://namez.com/profiles/1720-micah-silverman/?autoPlay=true>

File: aio/content/tutorial/toh-pt2.md (hunk 1 above)
Comment (dogeared) [reply aio/content/tutorial/toh-pt2.md]: I updated the PR per your suggestions.

**Comment by LinBoLen** [comment]
can words be a little bit softer?

**Review by kapunahelewong — APPROVED** [review]
LGTM

**Comment by gkalpak** [comment]
> In other places, the tutorial is explicit about where to place things as it makes it more clear. For instance: https://angular.io/tutorial/toh-pt1#hero-object
> It's not technically critical that the new Hero class goes right below the import statement.

It is actually critical. Since that file only contains `import` statements and the `AppComponent` class, and since classes are not hoisted (like function), it is important to put the `Hero` class before the `AppComponent` class (and below the `import` statements, which according to the spec should be at the top).

The new wording is fne by me. Just for the recond, imo making it sound like there are more limitations than necessary tends to confuse newcomers down the road.

:+1: for expanding the docregions. In fact, I think it might be useful to introduce a new feature for context in docregions that show adding to or modifying an existing file. I.e. 1-2 lines above and below the affected lines would appear grayed-out (to show what was there before the change), while the affected lines would be highlighted in color as usual.

**Comment by jenniferfell** [comment]
@dogeared : Hi. First, let me apologize for this sitting so long. I agree that this change is very helpful to readers. Would you be willing to resolve the merge conflicts? We'd be super appreciative, and if you tag me when you're done, I'll make sure to get it merged. :-)

**Comment by jenniferfell** [comment]
@dogeared Hi. I investigated this some more. It's still a great idea, and I like your wording. Things have moved around a bit since you submitted this PR. We're happy to incorporate your change into another PR we're working on (https://github.com/angular/angular/pull/24445). Or...if you'd prefer to update this PR, we'll help you move it forward this time. Thanks again for this submission, and making us aware of a general theme we should be better about: where to do things. Thanks again.

**Comment by jenniferfell** [comment]
Also, I opened a new issue for the larger question of how can we show more context in code regions. Thx also to @gkalpak

**Comment by brandonroberts** [comment]
Closing as no longer applicable

**Pull request closed without merging**